pkg/sample-plugin.go

```diff
@@ -80,8 +79,7 @@ func (td *SampleDatasource) query(ctx context.Context, query backend.DataQuery)
 	// Return an error is `Format` iis empty. Returning an error on the `DataResponse`
 	// will allow others queries to be executed.
 	if qm.Format == "" {
-		response.Error = errors.New("format cannot be empty")
-		return response
+		log.DefaultLogger.Warn("format is empty. defaulting to time series")
```

Comment:
Can we update the comment here, since it's wrong with regards to the new behaviour?

Reply:
Fixed